.github/workflows/publish.yml

```diff
@@ -5,15 +5,14 @@ on:
 jobs:
   build_current:
     name: Build current
-    if: github.ref == 'refs/heads/master'
```

Comment:
This reverts commit ed94fa4f74eae7a5486892375aa87f13270f9dd8 - is this intentional?

Reply:
@seabass-labrax 
Yes. 
https://github.com/spdx/spdx-spec/blob/development/v2.3/.github/workflows/publish.yml#L49
In line 49, It was very difficult for me to write logic to determine whether a build of the master branch succeeded if it was committed/pushed to the master branch, and whether a build of the v 2.3 branch succeeded if it was committed/pushed to the v2.3 branch. Therefore, I took the approach (reverted) that if a modification was committed/pushed to either branch, I would build both and reconfigure the website.
However, since I have listed the dependent pip packages in publish.yml, the only file we need to modify for future library updates is publish.yml in the default branch.